### ISSUE TITLE
Make the "zen mode" button minimally findable

### ISSUE THREAD

**Issue by rakslice** [elk-zone/elk#1883]
It took me a while to figure out why left side main menu bar (Home, Notifications, Conversations, etc.) started fading out when I'm not actively pointing at it, with no indication that there is anything available by pointing there while it is hidden, for no reason as far as I could tell as it just leaves that area of the window as unused wasted space -- but after searching for the missing docs, scouring all the preferences, and fiddling around with the window size, I started pointing at random parts of the window and I finally noticed the fine print nav on the other side that also only appears when you're pointing at it has buttons on it, and this is apparently a feature, not a bug: "Zen Mode".

Okayyyy.

Could you maybe:
- Make the Zen Mode setting also appear in the place in that people are rationally going to look for a setting like that, which is the Interface section of the Settings, like you already do for the Toggle Theme button right next to it
- Better yet make the things that disappear in Zen Mode when hidden show some minimal indication that there's something to find there

**Comment by rtxanson** [comment]
I noticed this because I accidentally hit the keyboard shortcut (`z`) that I didn't know about (and it must have been before switching tabs, because I don't know when I hit it-- unless it comes on by default after upgrades), and then couldn't find a place to disable the keyboard shortcuts or read about them.

**Comment by ayoayco** [comment]
This was not a problem before because you would never toggle zen mode without knowing where the icon is. But we just recently added the hotkey ‘z’ for Zen Mode and that will indeed be easy to accidentally hit.

I think the proposed solution to add Zen Mode in the settings is good. And maybe under Preferences > Wellbeing is a good option as well.

**Comment by rtxanson** [comment]
I noticed the mobile version has zen mode on the menu along with the local and federated timelines and etc., so a sidebar button could be particularly useful on desktop too.

**Comment by EddiesTech** [comment]
Is there any reason `z` was chosen, as opposed to something like `Alt`+`z`. I imagine this is something you would choose once and not go back and forth on, so why make it such an easy shortcut? Why have a shortcut at all? I imagine there is a reason, but I can't find it

**Comment by shaunagm** [comment]
I'm another person who must have accidentally hit the Z button. After a day of thinking "maybe Elk is having performance problems" and using a different interface, I spent a little while looking through the settings before eventually ending up here. Since it took me almost a day to have the bright idea to check the settings, I would also recommend also making the short key a little harder to accidentally hit, in addition to exposing this in the settings.

**Comment by ayoayco** [comment]
The hotkeys are still fresh and I believe because Zen Mode is a unique Elk feature that is not in the default Mastodon Web app, we are still figuring things out and we do appreciate the feedback! (After all, we are still in alpha) 😁

So to clarify, I see too suggestions here:
1. make the hotkey to toggle Zen Mode more intentional, so add some modifier keys instead of just 'Z'
2. add a setting, possibly under Preferences

**Comment by scottjenson** [comment]
Zen mode feels like a setting, not something you'll toggle repeatedly so the need for a 'quick toggle button' seems low? So many people appear to be have affected by this and it's not clear what the value is to the keyboard shortcut. If you were to remove the shortcut key, this problem would go away.

However, Like everyone else, I went to settings to reverse it and couldn't find it. That seems like the cleanest solution, move the setting from a corner of the screen into the place where people will actually look to find things like this.

**Comment by manuq** [comment]
> Zen mode feels like a setting, not something you'll toggle repeatedly so the need for a 'quick toggle button' seems low? So many people appear to be have affected by this and it's not clear what the value is to the keyboard shortcut. If you were to remove the shortcut key, this problem would go away.
> 
> However, Like everyone else, I went to settings to reverse it and couldn't find it. That seems like the cleanest solution, move the setting from a corner of the screen into the place where people will actually look to find things like this.

I'm not sure if the shortcut should be removed or not but I second that: Zen mode should be in settings. Maybe under "Wellbeing". With a good explanation of what it does, like the description under "Hide username emojis".

**Comment by scottjenson** [comment]
Is there a use case when you'll toggle Zen mode on and off repeatedly? I just don't see anyone needing this (please correct me if I'm wrong) This is a something you set and forget. Adding a shortcut key with no real need is just a waste of the programmer's time and as we've seen, causes some real user problems.

Let me put it this way. Are there shortcuts for EVERYTHING in settings? Of course not. So why is Zen mode special?

**Comment by EddiesTech** [comment]
Agreed. This is exactly what I said above.

**Comment by manuq** [comment]
> Is there a use case when you'll toggle Zen mode on and off repeatedly?

Zen is in some editors a temporary mode that you enable when you want to focus. But this is a social media app, I agree it doesn't need a shortcut.

**Comment by patak-dev** [comment]
I think the original idea was exactly as @manuq said, being able to focus on reading the timeline for a while. @antfu @ayoayco is this the way you are using it? Or do you have zen mode turned on all the time? (I don't use zen mode myself to be honest)

**Comment by antfu** [comment]
Until we make the shortcut customizable, I think we could remove the shortcut for Zen mode, as I don't see the need to toggle that often either.

**Comment by manuq** [comment]
Then can we agree on?
- Add a new zen mode setting (AFAIK there isn't any) 
- remove shortcut for now

**Comment by patak-dev** [comment]
@manuq yes, I think that sounds good. PR Welcome!
